presto-cache/src/main/java/com/facebook/presto/cache/filemerge/FileMergeCacheManager.java

```diff
@@ -407,7 +407,7 @@ private boolean write(FileReadRequest key, byte[] data, Path newFilePath)
 
         // no lock is needed for the following operation
         if (updated) {
-            // remove the the previous or following file as well
+            // remove the previous or following file as well
```

Comment:
This is flagged by special tooling. I also am using code to apply fixes, so I haven't necessarily reviewed each change, but I'm relatively confident of most of them.